tests/dep_test_permutation.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Testing File
+"""
+# Import necessary modules
+import numpy as np
+import pyrft as pr
+import os
+
+# The location where the pyrft module is saved (need to specify yourself)
+pyrft_loc = 'C:/Users/12SDa/davenpor/davenpor/Toolboxes/pyrft/'
+
+# Where data is stored
+data_store = pyrft_loc + 'tests2/test_data/'
+
+def test_boot_contrasts():
+    np.random.seed(10)

```

Comment:
A small docstring is always welcome to explain what is tested here

Reply:
I'm going to resolve the comments about dep_test_permutation because I have tried to make things in it align with standards (I'm just keeping it in case I need to use something in that file at a future date).

Reply:
I meant to say I have not tried

Reply:
Would you prefer that I create a separate folder for deprecated functions?

Reply:
No, remove them. Anything that you don't remove now will never be removed and make you codebase a mess.